data/src/main/java/org/apache/iceberg/data/TableScanIterable.java

```diff
@@ -80,15 +80,14 @@
 
   private CloseableIterable<Record> open(FileScanTask task) {
     InputFile input = ops.io().newInputFile(task.file().path().toString());
-    Map<Integer, ?> partition = PartitionUtil.constantsMap(task, TableScanIterable::convertConstant);
```

Comment:
Is the rename needed? This could cause commit conflicts.

Reply:
fixed